src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/services/AssessmentNeedsService.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.hmpps.assessrisksandneeds.services
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.AssessmentNeedDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.AssessmentNeedsDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.OffenderNeedDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.OffenderNeedsDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.restclient.AssessmentApiRestClient
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.services.exceptions.EntityNotFoundException
+
+@Service
+class AssessmentNeedsService(private val assessmentClient: AssessmentApiRestClient) {
```

Comment:
Could we please add a test for the service layer? this is easily unit tested. imo we should prioritise unit testing over Integration testing because the upper the layer the longer the test takes to run. IT testing should cover only one the most common paths and the rest should be down to unit testing layer.